### PR TITLE
Allow special character in MongoDB password

### PR DESCRIPTION
**Pull request by ebyhr**
Fix https://github.com/prestodb/presto/issues/13048

### CODE REVIEW

File: presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java [hunk 1]
```diff
@@ -35,7 +35,6 @@
 {
```
Comment (kokosing) [inline presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java]: can you please provide a test for that?

Comment (ebyhr) [reply presto-mongodb/src/main/java/io/prestosql/plugin/mongodb/MongoClientConfig.java]: Added TestMongoClientConfig.java

**Comment by ebyhr** [comment]
Sorry, I missed existing code has `trimResults().omitEmptyStrings()`.  My commit may break existing configuration. Let me fix tomorrow.

**Comment by findepi** [comment]
@ebyhr i noticed Maria's comment about trimming. However, I couldn't come up with a situation where we actually want the trimming, therefore I approved as-is.
Still, I am open for your suggestions.

**Comment by ebyhr** [comment]
@findepi Now, I think so. I did additional tests on local, it looks fine.

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**